bots/npm-update

```diff
@@ -136,5 +135,8 @@ def run(package, verbose=False, **kwargs):
                 task.comment(pull, comment)
                 break
 
+    # Undo our changes
+    package_json(orig_package_json)
```

Comment:
That is easier to understand, thanks! However, what is this line for? Just for being able to run this script locally several times, without having to manually reset?

Reply:
Yes, exactly.